src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,73 @@
 // your components will all go in this `component` directory.
 // feel free to change this component.js into TodoList.js
+import React from "react";
+import Todo from "./Todo";
+
+const todos = [
+  {
+    task: "Testomo",
+    id: Date.now(),
+    completed: false
+  },
+  {
+    task: "Testo2",
+    id: Date.now(),
+    completed: false
+  },
+  {
+    task: "Testo3",
+    id: Date.now(),
```

Comment:
though I don't recommend you using date.now I think it should be fine for this project. Keep in mind that you can use other values or just find a way to increment the number of the id by 1. (this will also create a bug if something is deleted as well but I think the chances of that bug showing up slightly less then the date objects)